docker/server/build.gradle

```diff
@@ -26,31 +26,24 @@ baseMap.keySet().each {base ->
     evaluationDependsOn Docker.registryProject(base)
 }
 
-evaluationDependsOn ':deephaven-jpy'
-evaluationDependsOn ':deephaven-wheel'
-evaluationDependsOn ':deephaven2-wheel'
-
 configurations {
     serverApplicationDist
+    pythonWheel
 }
 
 dependencies {
     serverApplicationDist project(path: ':server-netty-app', configuration: 'applicationDist')
+
+    pythonWheel project(path: ':deephaven-jpy', targetConfiguration: 'pythonWheel')
```

Comment:
Technically this isn't required any more, it is transitively picked up by py-server and deephaven-wheel both - but since we expect people will actually directly call `jpy.get_type` and such, I elected to explicitly list it. Thoughts?

Reply:
My preference would be to leave it out, unless we have code in `docker/server` (project) that refers to `jpy`.